.github/workflows/docker-build-and-push-humble-self-hosted.yaml

```
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - v*
+      - \d+.\d+
```

I'm not sure, but maybe this is better?

https://docs.github.com/ja/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

```suggestion
      - [0-9]+.[0-9]+*
```

---

Fixed.
FYI, it seems we need to have quotes `'` for the keys that start with `[`

> The characters *, [, and ! are special characters in YAML. If you start a pattern with *, [, or !, you must enclose the pattern in quotes.

---

I see, thank you!